storybookjs/storybook
# Add PX Blue to examples page

## What I did
Added Power Xpert Blue to the examples page in the docs

## How to test
- Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.
